DemoServer/Controllers/Demos/Attachments/IndexAttachmentDetails/metadata.json

```diff
@@ -0,0 +1,160 @@
+{
+    "Slug": "index-attachment-details",
+    "SourceFileName": "IndexAttachmentDetailsController.cs",
+    "Title": "Index Attachment Details",
+    "DescriptionLines": [
+        "**attachment's details and content can both be indexed**.",
```

Comment:
First letter of this sentence is lower-case

Reply:
done